.changes/unreleased/tinymce-TINY-11211-2024-09-10.yaml

```diff
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Added
+body: Buttons can now be enabled or disabled based on whether their context matches a given predicate, with state updates on `init`, `NodeChange`, and `SwitchMode` events.
```

Comment:
Maybe we need to add a changelog about the new `context` property that all UI elements have. Not sure if that is a separate changelog item.

Reply:
yes we need one here, probably in this entry, the `bridge` changelog isn't published to customers. It will also need a docs ticket.

Reply:
Oops, missed that. I've included that in this entry. Could you please take a look and let me know if it works?

Reply:
much better, thank you, just suggested some tweaks :)